rebasehelper/application.py

```diff
@@ -217,12 +217,16 @@ def _initialize_data(self):
         self.old_rest_sources = [os.path.abspath(x) for x in self.spec_file.get_sources()[1:]]
         self.new_rest_sources = [os.path.abspath(x) for x in self.rebase_spec_file.get_sources()[1:]]
 
-    def get_rpm_packages(self, dirname):
-        """
-        Function returns RPM packages stored in dirname/old and dirname/new directories
+    def get_rpm_packages(self, dirname: str) -> bool:
```

Comment:
The method name also doesn't make much sense.

Reply:
So what name do you suggest? Perhaps `check_rpm_packages` or `find_rpm_packages`?

Reply:
Well, it's hard to say, because the method does more than just checking if there are some RPMs, but something like `update_build_data` doesn't sound right either.

Reply:
It is probably impossible to find a short and descriptive name for this method. I fee like `check` is the closest we may get as it describes the use-case more than `update_build_data`. Both are better than the current name

Reply:
Ok.